docs/components/tabs.md

```diff
@@ -297,6 +268,43 @@ const tabs = ref<Tab[]>([
 </script>
 ```
 
+## Usage
+
+### Tab links
+
+Passing `to` property for each tab object allows to render tabs as links. If a string is passed, it will be used in `href` attribute in the rendered `a` element. If an object is passed, the tab will be rendered as a `router-link`.
```

Comment:
Can you reword to explain this is an example of utilizing the `to` property along with the `hidePanels` prop?

The way it's currently written it sounds like you **must** use as links

Reply:
Rewritten